crates/driver/src/infra/config/file/load.rs

```diff
@@ -47,41 +44,7 @@ pub async fn load(path: &Path) -> infra::Config {
             }
         }))
         .await,
-        liquidity: liquidity::Config {
-            base_tokens: config
-                .liquidity
-                .base_tokens
-                .into_iter()
-                .map(eth::TokenAddress::from)
-                .collect(),
-            uniswap_v2: config
-                .liquidity
-                .uniswap_v2
-                .into_iter()
-                .map(|config| liquidity::config::UniswapV2 {
-                    router: config.router.into(),
-                    pool_code: config.pool_code.into(),
-                })
-                .collect(),
-            swapr: config
-                .liquidity
-                .swapr
-                .into_iter()
-                .map(|config| liquidity::config::UniswapV2 {
-                    router: config.router.into(),
-                    pool_code: config.pool_code.into(),
-                })
-                .collect(),
-            uniswap_v3: config
-                .liquidity
-                .uniswap_v3
-                .into_iter()
-                .map(|config| liquidity::config::UniswapV3 {
-                    router: config.router.into(),
-                    max_pools_to_initialize: config.max_pools_to_initialize,
-                })
-                .collect(),
-        },
+        liquidity: config.liquidity,
```

Comment:
...and the mapping from the config file representation (`LiquidityConfig`) into `liquidity::Config` should happen here, rather than in `run.rs`.